### DISCUSSION TITLE
Github repo doesn't update changes

### DISCUSSION THREAD

**Discussion by bogdanbgd7** [community/community#23690] [New to GitHub]
<p>Hello everyone, first of all, I need to say that I’m very glad to be part of the community!</p>
<p>So, I am very inexperienced when it comes to using github and today I started with new iOS project on which I wanted to learn github basics.</p>
<p>What I have done:</p>
<ol>
<li>Created repo on github</li>
</ol>
<ol start="2">
<li>
</li>
</ol>
<pre><code class="lang-auto">git init
git add .
git commit -m "my first commit"
git push -u origin master
</code></pre>
<p>After that, I’ve added some files and folders and did following:</p>
<pre><code class="lang-auto">git add .
git commit -m "added folders"
git push -u origin master
</code></pre>
<p>I looked at my github repo, and none of these changes had been made(neither folders nor files were added).<br>
What am I doing wrong?</p>
<p>Thanks in advance! :pray:</p>

**Comment by airtower-luna** [comment]
<p>I’m not seeing any command where you configure the <code>origin</code> remote. Unless you forgot that when describing what you did git wouldn’t know where to push, which would explain why your data isn’t showing up.</p>
<p>You can add the remote with a command like this:</p>
<pre><code class="lang-auto">git remote add origin URL
</code></pre>
<p>Where the URL for a repository on Github is something like <code>git@github.com:user/repository.git</code> for SSH or <code>https://github.com/user/repository.git</code> for HTTPS (replace “user” and “repository” with your data).</p>
<p>If that’s not the problem, could you share the output of the push commands so we can get a better idea of what’s going on?</p>

**Comment by bogdanbgd7** [comment, marked as answer]
<p>Thanks for reply!</p>
<p>I came across solution - branch was on “main” not on master.</p>
